### ISSUE TITLE
Transfer lua API manual to markdown

### ISSUE THREAD

**Issue by elstoc** [elstoc/dtdocs#86]
@wpferguson is looking into this.

**Comment by wpferguson** [comment]
I've looked at the generated website and I've forked the repo.  I do have a question.  It looks like the user manual is the top level document.  How are we going to make the API manual a separate document? Do we need to have a top level where you choose user manual or API manual?

**Comment by elstoc** [comment]
@wpferguson over to you to maintain in luadocs. Let me or @paperdigits know if we can help in any way.

**Comment by wpferguson** [comment]
Thanks for the support.  I did my "training wheel" projects last night and built a lua scripts manual with pages built from the script headers and a lua scripts api manual built from the wiki files which were already in markdown.  They both need some love (lots of proofreading), but they are up and running on my computer.  

I tried a couple of quick and dirty translators on the API manual, but didn't have much luck.  It's now a ruby/nokogiri project :).
